files/__addonLocation__/unpublished-development-types/index.d.ts

```diff
@@ -4,11 +4,11 @@
 import '@glint/environment-ember-loose';
 import '@glint/environment-ember-template-imports';
 
-declare module '@glint/environment-ember-loose/registry' {
```

Comment:
instead of adding new lint disables here, I've just commented out the whole thing